torch/csrc/jit/autodiff.cpp

```diff
@@ -148,10 +95,10 @@ bool isDifferentiable(Node* n) {
     return n->get<std::vector<int64_t>>(attr::size) &&
         n->namedInput(attr::self)->type()->cast<CompleteTensorType>();
   }
-  if (n->matches(
-          "aten::nll_loss(Tensor self, Tensor target, Tensor? weight, int reduction, int ignore_index) -> Tensor")) {
-    // TODO(asuhan): support weight
-    return n->namedInput(attr::weight)->node()->mustBeNone();
+
```

Comment:
What happened to the nll_loss code here?

Reply:
For nll_loss, expand, squeeze we have added them in previous PRs but didn't remove them here. :P 
For tensor ops these formulas get picked up automatically and tested in our test_jit, if there's an error in implementation, it will throw :D 
Nice catch about nn ops! We need to enable them specifically, I will make a follow up commit to enable them here.

Reply:
Oh actually I'm wrong, those nn tests are already enabled when we added the AD formula in symbolic variable. So we should be good on those as well. ;)